core/src/main/scala/cromwell/core/KnowsWhatTimeItIs.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.core
+
+import java.sql.Timestamp
+import java.util.Date
+
+
```

Comment:
While this made me laugh I'd prefer something a little less tomfooleryish. OTOH I didn't really have a better suggestion in the more serious category. At least you didn't call it FlavaFlav I suppose :)

![flavaflav_main1](https://cloud.githubusercontent.com/assets/961771/15373325/164c5670-1d13-11e6-84b1-43b334b9bdc1.jpg)